low-level/files/opt/seagate/sspl/setup/manifest_support_bundle

```diff
@@ -65,42 +64,42 @@ class ManifestSupportBundle(object):
 
 
     def make_manifest_support_bundle_tarfile(self, parser, tmpFile):
+        localTempPath = "/tmp/support_bundle/"
```

Comment:
not supposed to use /tmp, default to sspl data folder, create appropriate folder under it and use

Reply:
Done